### PR TITLE
Fixes #26015 Update docker plugin installation

### PR DESCRIPTION
**Pull request by nodermatt**
Updated instructions to install foreman docker plugin and updated screenshot

### CODE REVIEW

File: plugins/foreman_docker/3.x/index.md [hunk 1]
```diff
@@ -227,9 +249,9 @@ Note that you can not change the container configuration once the container is c
 ![Container wizard config](static/images/plugins/foreman_docker/docker-config-options.png)
 
 5. In the final stage of container creation named Environment, select if you want to allocate a pseudo-tty, attach STDIN, STDOUT, and STDERR to the container. Click Add environment variable to create a custom environment variable for the container.
-![Container wizard environment](static/images/plugins/foreman_docker/docker-environment.png)
+![Container wizard environment](static/images/plugins/foreman_docker/docker-environment.png) - Don't forget to select the checkbox "run". Otherwise the newly created container will be in "EXITED" state after creation.
 
-6. Click Submit to create the container.
+7. Click Submit to create the container.
```
Comment (evgeni) [inline plugins/foreman_docker/3.x/index.md:254]: Why did the numbering change? The previous step is still 5?

Comment (nodermatt) [reply plugins/foreman_docker/3.x/index.md:254]: This is a mistake, it shouldn't be step no 7 of course.

Good catch +1

File: plugins/foreman_docker/3.x/index.md [hunk 2]
```diff
@@ -162,9 +162,18 @@ In Foreman, you can deploy containers only on a compute resource of the Docker p
...
+
+      ```
+      $ sudo mkdir -p /etc/systemd/system/docker.service.d/
+      $ sudo vi /etc/systemd/system/docker.service.d/startup_options.conf
+      ```
+      Now paste the below content into the created file.
+      ```
+      [Service]
+      ExecStart=
+      ExecStart=/usr/bin/dockerd -H fd:// -H tcp://0.0.0.0w:[PORT_NUMBER]
```
Comment (ekohl) [inline plugins/foreman_docker/3.x/index.md:175]: Is the `w` supposed to be in there?

Comment (nodermatt) [reply plugins/foreman_docker/3.x/index.md:175]: Hi @ekohl, no this typo probably came from when I wanted to write the file in vi and hadn't left the insert mode yet. Will correct this.

**Comment by tbrisker** [comment]
@nodermatt Thank you for opening the PR! looks good to merge other than the two typos, can you please fix them and squash all the commits into one?

**Comment by tbrisker** [comment]
Fixed on merge, thanks @nodermatt ! merged as f6bdeba4c02334a573a26b38bb55e8fe8d44e182.

**Pull request closed without merging**